_data/tables/transform-high-performing-org.yml

```diff
@@ -39,22 +44,60 @@
       data:
         list: false
         fr: 
-        en: Establish communication plan and change management for continuous improvement in (and around) IITB
+        en: Develop a self assessement tool based on capabilities of high performing organizations and PwC recommendations
     - col:
       data:
         list: false
         fr: 
-        en: 
+        en: To help teams understand the current condition and set measurable targets, the self assessment tool will guide teams toward ways to improve based on an evaluation of diferent capabilities linked to high performing organizations.
     - col:
       data:
         list: true
         items:
         - item:
-            fr: <strong>BOP</strong>
-            en: <strong>BOP</strong>
+            fr: <strong>Stratégie TI</strong>
+            en: <strong>IT Strategy</strong>
+        - item:
+            fr: R&P
+            en: R&P
+    - col:
+      data:
+        list: false
+        fr: 2020-2021 Q2
+        en: 2020-2021 Q2
+  - row:
+    headers: false
+    cols:
+    - col:
+      data:
+        list: false
+        fr: Gouvernance
+        en: Governance
+    - col:
+      data:
+        list: false
+        fr: 
+        en: Define how DGs will guide their teams to achieve the goals of the Guiding Policy
+    - col:
+      data:
+        list: false
+        fr: 
+        en: The goals of the Guiding Policy is to move towards a Generative (Westrum) culture; to reduce lead time, change failure rate and time to restore service; while increassing deployement frequency. DGs will need to identify how their teams can contribute to the improvement of these metrics, by defining and improving their own metrics.
```

Comment:
I think we can remove this one. It is a desired outcome, not so much an action in and of itself